Data/Vector/Generic.hs

```diff
@@ -203,6 +203,8 @@ import Prelude hiding ( length, null,
 import qualified Text.Read as Read
 import qualified Data.List.NonEmpty as NonEmpty
 
+import qualified Data.Traversable as T (Traversable(mapM))
+
```

Comment:
was this shuffled  because of how you wanted to do the cpp to conditionalize mkNoRepType?

Reply:
This was shuffled because function definitions should not appear between imports, if I didn't move it it would look like this for `base < 4.2.0`:
```haskell
import Data.Data ( Data, DataType, Constr, Fixity(Prefix),
                   mkDataType, mkConstr, constrIndex )
                   mkDataType, mkConstr, constrIndex,
                   mkNorepType )
mkNoRepType :: String -> DataType
mkNoRepType = mkNorepType

import qualified Data.Traversable as T (Traversable(mapM))
```

The slight improvement to the way I did `CPP` was to reduce multiple imports from the same module, but it had nothing to do with `Traversable` import. `hlint` is a great tool that points those flaws to you.